schematics/collection.json

```diff
@@ -0,0 +1,18 @@
+// By default, collection.json is a Loose-format JSON5 format, which means it's loaded using a
```

Comment:
Add comment at the top that explains what this file is for / does?

Reply:
Technically, json is not supposed to have comments. ;P

Reply:
It has comments already?